src/Service/Context/PageContext.php

```diff
@@ -173,9 +177,20 @@ private function setContextByNode(Request $request) {
    *   The route object.
    * @param \Drupal\Core\Controller\TitleResolverInterface $titleResolver
    *   The title resolver.
+   * @param Drupal\Core\Language\LanguageManagerInterface $languageManager
+   *   The language manager.
    */
-  private function setContextTitle(Request $request, Route $route, TitleResolverInterface $titleResolver) {
-    // Find title.
+  private function setContextTitle(Request $request, Route $route, TitleResolverInterface $titleResolver, LanguageManagerInterface $languageManager) {
+    // Set language code
+    // After investigation, there is no use case where the methods
+    // 'getCurrentLanguage' and 'getId' would not exist within 
+    // LanguageManager. Drupal 8 will ALWAYS set a language code
+    // and would not be null, therefore no checks are required
+    // @link https://api.drupal.org/api/drupal/core%21lib%21Drupal%21Core%21Language%21LanguageManager.php/class/LanguageManager/8.6.x
+    // @link https://api.drupal.org/api/drupal/core%21lib%21Drupal%21Core%21Language%21Language.php/class/Language/8.2.x
```

Comment:
I personally don't prefer linking in code but I wouldn't be against it later.

Reply:
Removed the links as they may 404/mislead in the future